content/blog/pki-maze.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Designing a Permanent Personal Identity"
+date = 2019-11-20
+description = "A public key infrastructure (PKI) is a system for binding a set of keys to a name. Sometimes a small amount of metadata is included."
```

Comment:
This text should also be in the text of the post, it's important for framing the rest of it.

Reply:
Ah, just fixed this — thanks for pointing it out.

Reply:
Come to think of it, for the "description" part, the second sentence is probably noise.